tests/ecosystem/upgrade/test_upgrade.py

```diff
@@ -1,66 +1,48 @@
 import logging
 
+import pytest
+
 from ocs_ci.framework.testlib import ocs_upgrade
-from ocs_ci.framework import config
-from ocs_ci.ocs.cluster import CephCluster, CephHealthMonitor
-from ocs_ci.ocs.resources.storage_cluster import (
-    ocs_install_verification
-)
-from ocs_ci.utility.utils import TimeoutSampler
-from ocs_ci.ocs.ocs_upgrade import (
-    verify_image_versions,
-    OCSUpgrade,
-)
-from ocs_ci.ocs.exceptions import TimeoutException
+from ocs_ci.ocs.disruptive_operations import worker_node_shutdown
+from ocs_ci.ocs.ocs_upgrade import run_ocs_upgrade
+
 
 log = logging.getLogger(__name__)
 
 
+@pytest.mark.parametrize(
+    argnames=["operation"], argvalues=[
+        pytest.param(
+            worker_node_shutdown, marks=pytest.mark.polarion_id("OCS-1579")
+        ),
+    ]
+)
+def test_worker_node_abrupt_shutdown(operation):
+    log.info("Starting disruptive test function: "
+             "test_worker_node_abrupt_shutdown")
+    abrupt = True
+    run_ocs_upgrade(operation, abrupt)
+
+
+@pytest.mark.parametrize(
+    argnames=["operation"], argvalues=[
+        pytest.param(
+            worker_node_shutdown, marks=pytest.mark.polarion_id("OCS-1575")
+        ),
+    ]
+)
+def test_worker_node_permanent_shutdown(operation):
+    log.info("Starting disruptive test function:"
+             " test_worker_node_permanent_shutdown")
+    abrupt = False
+    run_ocs_upgrade(operation, abrupt)
```

Comment:
```suggestion
    run_ocs_upgrade(operation=worker_node_shutdown, abrupt=False)
```

The parametrize is bringing here the complexity which we don't need at all I think.

Reply:
removed